custom/icds_reports/utils/data_pull/Oct-2019.sql

```diff
@@ -0,0 +1,126 @@
+awc_days_open,
+CASE WHEN num_launched_awcs>0 THEN awc_days_open/num_launched_awcs ELSE awc_days_open END AS average_awc_open
+FROM agg_awc_monthly WHERE aggregation_level=1 AND month='2019-10-01'
+
+
+-- # of AWCs that submitted Infra form
+SELECT state_name, sum(num_awc_infra_last_update)
+FROM agg_awc_monthly
+WHERE aggregation_level=1 AND month='2019-10-01'
+GROUP BY state_name
+
+/*
+# of AWCs that reported available drinking water
+# of AWCs that reported unavailable drinking water
+# of AWCs that reported available functional toilet
+# of AWCs that reported unavailable functional toilet
+*/
+SELECT state_name,
+count(*) FILTER (WHERE infra_clean_water IN (1,2,3,5)) AS "Available drinking water",
+count(*) FILTER (WHERE infra_clean_water=0) AS "Unavailable drinking water",
+count(*) FILTER (WHERE infra_functional_toilet=1) AS "Available functional toilet",
+count(*) FILTER (WHERE infra_functional_toilet=0) AS "Unavailable functional toilet"
+FROM agg_awc_monthly
+WHERE aggregation_level=5 AND month='2019-10-01'
+GROUP BY state_name
+
+/*
+# of AWCs that reported usable infantometer
+# of AWCs that reported unavailable usable infantometer
+# of AWCs that reported usable stadiometer
+# of AWCs that reported unavailable usable stadiometer
+*/
+SELECT state_name,
+count(*) FILTER (WHERE infantometer=1) AS "AWCs that reported usable infantometer",
+count(*) FILTER (WHERE infantometer=0) AS "AWCs that reported unavailable usable infantometer",
+count(*) FILTER (WHERE stadiometer=1) AS "AWCs that reported usable stadiometer",
+count(*) FILTER (WHERE stadiometer=0) AS "AWCs that reported unavailable usable stadiometer"
+FROM "public"."awc_location_months_local" "awc_location_months" LEFT JOIN "public"."agg_awc" "agg_awc" ON (
+        ("awc_location_months"."month" = "agg_awc"."month") AND
+        ("awc_location_months"."aggregation_level" = "agg_awc"."aggregation_level") AND
+        ("awc_location_months"."state_id" = "agg_awc"."state_id") AND
+        ("awc_location_months"."district_id" = "agg_awc"."district_id") AND
+        ("awc_location_months"."block_id" = "agg_awc"."block_id") AND
+        ("awc_location_months"."supervisor_id" = "agg_awc"."supervisor_id") AND
+        ("awc_location_months"."awc_id" = "agg_awc"."awc_id")
+    )
+WHERE "agg_awc"."aggregation_level"=5 AND "agg_awc"."month"='2019-10-01'
+GROUP BY state_name
+
+
+/*
+# of AWCs that reported available medicine kit
+# of AWCs that reported unavailable medicine kit
+# of AWCs that reported available infant weighing scale
+# of AWCs that reported unavailable infant weighing scale
+# of AWCs that reported available mother and child weighing scale
+# of AWCs that reported unavailable mother and child weighing scale
+*/
+SELECT state_name,
+count(*) FILTER (WHERE infra_medicine_kits=1) AS "Available medicine kit",
+count(*) FILTER (WHERE infra_medicine_kits=0) AS "Unavailable medicine kit",
+count(*) FILTER (WHERE infra_baby_weighing_scale=1) AS "Available infant weighing scale",
```

Comment:
hey this property will be used 'infra_infant_weighing_scale'

Reply:
oops, okay

Reply:
https://github.com/dimagi/commcare-hq/pull/25893/commits/8233a57d107678ccc8e3c4cd86bf77a5d219f974